src/index.css

```
@@ -20,6 +20,12 @@ body {
     @apply antialiased text-gray-900 bg-white;
     font-feature-settings: 'cv11', 'ss01';
   }
+
+  :focus-visible {
+    outline: 2px solid currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)